cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,272 @@
+/// <reference types="Cypress" />
```

Comment:
should these example files go in .gitignore too?

Reply:
its done